wercker.yml

```diff
@@ -4,22 +4,14 @@ build:
     # Install package
     - script:
       name: install package
-      code: pip install .[dev,plotly,server]
-
-    # Run formatters
-    - script:
-      name: run formatters
-      code: |
-        isort --check-only
-        black --check .
+      code: make install
```

Comment:
just nitpicking: separate steps for black, flake, isort and the like would give more detailed information about the actual step that caused the failure

Reply:
i did it this way because its a single command and make outputs a command before execution so we should still be able to see the precise command that failed